src/ValidatedInputTrait.php

```diff
@@ -32,4 +32,9 @@ public function getValidationResult(): Result
 
         return $this->validationResult;
     }
+
+    public function resetValidation(): void
```

Comment:
```suggestion
    public function resetResult(): void
```

Reply:
May be `resetValidationResult()` better?